.ci.yaml

```diff
@@ -156,6 +156,19 @@ targets:
       emulator_arch: "x64"
     timeout: 60
 
+  - name: Linux Fuchsia FEMU (cso)
+    recipe: engine/femu_test
+    bringup: true
+    properties:
+      add_recipes_cq: "true"
```

Comment:
False? as this build will be removing caches it will slow down recipes testing. Femu_test recipe is already being covered by the other builders.

Reply:
You tell me =) I have no idea what the flag is supposed to mean. I just copy-pasted from the other config.